server.py

```diff
@@ -27,6 +30,26 @@ def setup_logger():
             logging.getLogger(logger_name).setLevel(100)
 
 
+def validate_environment():
+    versions = {
```

Comment:
yikes

Reply:
will this work on Windows? i think we can find a more simplistic way. we should also store the version in default.yml.

Reply:
yea, I was wondering what we wanted to do with windows, good you brought it up, since install.sh doesnt even cover windows, so I was not too sure how people were using it there.

Reply:
and if the versions are inherent to the code, should it be in default.yml where someone could change it?

Reply:
for windows, you're right, we don't support it. so probably not worth bending over backwards on this check for it. in terms of where the versions should be, yeah, they should be in a config file: your argument could be applied to anything (well, that password is used in code...) :)